core/model/src/main/java/org/eclipse/rdf4j/model/impl/AbstractValueFactory.java

```diff
@@ -39,7 +39,7 @@
  *             {@link org.eclipse.rdf4j.model.base.AbstractValueFactory}.
  */
 @Deprecated
-public abstract class AbstractValueFactory implements ValueFactory {
+public abstract class AbstractValueFactory extends org.eclipse.rdf4j.model.base.AbstractValueFactory {
```

Comment:
I actually need to look at this, see #2655. I will likely refactor SimpleValueFactory to directly inherit from the new AbstractValueFactory, copying all the old AbstractValueFactory code over and leaving it orphaned.

Reply:
> I actually need to look at this, see #2655. I will likely refactor SimpleValueFactory to directly inherit from the new AbstractValueFactory, copying all the old AbstractValueFactory code over and leaving it orphaned.

As long as SimpleValueFactory inherits from the new AbstractValueFactor tests won't break.

Reply:
I've merged my update for #2655. It shouldn't break anything for you but might be worth rebasing, just to make sure.

Reply:
Rebased